src/hww.c

```diff
@@ -28,14 +75,32 @@
  */
 static void _msg(const Packet* in_packet, Packet* out_packet, const size_t max_out_len)
 {
+    if (in_packet->len >= 1) {
+        switch (in_packet->data_addr[0]) {
+        case OP_ATTESTATION:
+            _api_attestation(in_packet, out_packet);
+            return;
+        case OP_UNLOCK:
+            workflow_unlock();
+            out_packet->len = 0;
+            return;
+        default:
+            break;
+        }
+    }
+
+    // No other message than the attestation and unlock calls shall pass until the device is
+    // unlocked or ready to be initialized.
+    if (memory_is_initialized() && !keystore_is_unlocked()) {
```

Comment:
IMHO: it is easier to read code if the boolean function is positive. `keystore_is_locked()` is easer to understand than `!keystore_is_unlocked()`.

Reply:
Done.